dotnet-docs/docker-installation.md

```diff
@@ -42,13 +56,19 @@ This article is a step-by-step tutorial on deploying Telerik Report Server for .
 1. Run the command `docker stack deploy -c docker-compose.yml report-server`.
 1. Navigate to `localhost:82` in the browser to open the Report Server Manager for .NET.
 
->important The first time you open the Report Server, you will to configure it as explained in the article [Application Startup]({%slug application-startup%}).
 
-### Creating new Server Agent
+>important The first time you open the Report Server, you need to configure it as explained in the article [Application Startup]({%slug application-startup%}).
+
+We recommend using the images we provide instead of building custom ones. The sample code in the following sections assumes you use the images we provide. If you prefer custom images, change the code as follows:
+
+* `image: progressofficial/telerik-reportserver-app:latest` to `image: telerik-report-server:local`
+* `image: progressofficial/telerik-reportserver-agent:latest` to `image: telerik-report-server-agent:local`
+
+## Creating a new Server Agent
 
 >note The Report Server Manager for .NET has to be fully set up before following the steps from this section.
```

Comment:
This should not be a note. This should be the concept of the section, and again why an additional agent is necessary, what are the benefits, that the encryption key should be the same, and how to obtain the current encryption key from the running instance.